store/src/app/products/components/cart/cart.component.html

```diff
@@ -0,0 +1,29 @@
+<ng-container *ngIf="products.length >= 1">
```

Comment:
Could be just `*ngIf="!!products.length"`

Also you can take a look on this [article](https://ultimatecourses.com/blog/angular-ngif-else-then) 
![image](https://user-images.githubusercontent.com/43652702/209469416-0a2bd1ee-7d0a-4335-b495-24d871f42595.png)

Reply:
done